x/treasury/internal/keeper/keeper.go

```diff
@@ -195,76 +175,149 @@ func (k Keeper) PeekTaxProceeds(ctx sdk.Context, epoch int64) (res sdk.Coins) {
 	return
 }
 
-// ClearTaxProceeds clears all tax proceeds
-func (k Keeper) ClearTaxProceeds(ctx sdk.Context) {
-	store := ctx.KVStore(k.storeKey)
-	iter := sdk.KVStorePrefixIterator(store, types.TaxProceedsKey)
-	defer iter.Close()
-	for ; iter.Valid(); iter.Next() {
-		store.Delete(iter.Key())
-	}
-}
-
-// RecordHistoricalIssuance updates epoch issuance from supply keeper (historical)
-func (k Keeper) RecordHistoricalIssuance(ctx sdk.Context) {
-	epoch := core.GetEpoch(ctx)
+// RecordEpochInitialIssuance updates epoch initial issuance from supply keeper
+func (k Keeper) RecordEpochInitialIssuance(ctx sdk.Context) {
 	totalCoins := k.supplyKeeper.GetSupply(ctx).GetTotal()
-	k.SetHistoricalIssuance(ctx, epoch, totalCoins)
+	k.SetEpochInitialIssuance(ctx, totalCoins)
 }
 
-// SetHistoricalIssuance stores epoch issuance
-func (k Keeper) SetHistoricalIssuance(ctx sdk.Context, epoch int64, issuance sdk.Coins) {
+// SetEpochInitialIssuance stores epoch initial issuance
```

Comment:
`RecordEpochInitialIssuance` and `SetEpochInitialIssuance` are synonymous - Record and Set are both WRITE references; confusing. Consider removing one of the two functions.

Reply:
@YunSuk-Yeo ping on this